.github/workflows/deploy.yml

```diff
@@ -3,146 +3,52 @@ name: Deploy to Gov PaaS
 on:
   workflow_dispatch:
   push:
-    # branches:
-    # - master
-    # - FS-389
-    # - FS-390
-
     paths-ignore:
       - '**/README.md'
 
 jobs:
-  testing:
+  test_and_deploy:
+    uses: communitiesuk/funding-design-service-workflows/.github/workflows/deploy.yml@main
+    with:
+      app_name: ${{ github.event.repository.name }}
+      api: true
+      postgres_unit_testing: true
+      db_name: fsd_assess_store_test
+      deploy_to_dev: true
```

Comment:
I think we need to make this `false` as it means we won't be able to iterate at all on migrations in a PR.

Reply:
will do in future PR